src/invidious/views/template.ecr

```diff
@@ -153,6 +153,11 @@
                         <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
                     </div>
                 </div>
+            <% if CONFIG.footer %>
+            <div class="h-box" style="padding-top: 1em; padding-bottom: 0.5em;">
+               <div id="footer-info" style="text-align: center;"><%= CONFIG.footer %></div>
```

Comment:
I'd avoid using inline CSS, that's why you added classes and/or IDs. Or you might as well add `v-box` class to the div and avoid specifying custom styles entirely. The `footer` HTML5 semantic element already has `text-align: center` set in  https://github.com/iv-org/invidious/blob/master/assets/css/default.css#LL358C3-L358C22, which should get inherited here.
Your indentation is wrong as well.